eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -14,49 +14,30 @@ param (
   $DocRepoContentLocation = "docs-ref-services/" # within the doc repo, where does our readme go?
 )
 
-. (Join-Path $PSScriptRoot artifact-metadata-parsing.ps1)
-. (Join-Path $PSScriptRoot SemVer.ps1)
+. (Join-Path $PSScriptRoot common.ps1)
 
 $releaseReplaceRegex = "(https://github.com/$RepoId/(?:blob|tree)/)master"
 
-function GetMetaData($lang){
-  switch ($lang) {
-    "java" {
-      $metadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/java-packages.csv"
-      break
-    }
-    ".net" {
-      $metadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/dotnet-packages.csv"
-      break
-    }
-    "python" {
-      $metadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
-      break
-    }
-    "javascript" {
-      $metadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
-      break
-    }
-    default {
-      Write-Host "Unrecognized Language: $language"
-      exit(1)
-    }
+function GetMetaData {
```

Comment:
I believe @sima-zhu has also defined this function somewhere we should probably move it to one our helper scripts so we can share it.